operators/audio/audio_decoder.hpp

```diff
@@ -98,16 +96,20 @@ struct KernelAudioDecoder : public BaseKernel {
     return total_buf_size;
   }
 
-  void Compute(OrtKernelContext* context) {
-    const OrtValue* input = ort_.KernelContext_GetInput(context, 0);
-    const uint8_t* p_data = ort_.GetTensorData<uint8_t>(input);
-
-    OrtTensorDimensions input_dim(ort_, input);
+  void Compute(const ortc::Tensor<uint8_t>& input,
+               const std::optional<std::string> format,
+               ortc::Tensor<float>& output0) {
+    const uint8_t* p_data = input.Data();
+    auto input_dim = input.Shape();
     if (!((input_dim.size() == 1) || (input_dim.size() == 2 && input_dim[0] == 1))) {
       ORTX_CXX_API_THROW("[AudioDecoder]: Expect input dimension [n] or [1,n].", ORT_INVALID_ARGUMENT);
     }
 
-    auto stream_format = ReadStreamFormat(context, p_data);
+    std::string str_format;
+    if (format.has_value()) {
+      str_format = format.value();
```

Comment:
`optional::value` is not available in some older MacOS/iOS versions. can either increase the target version or just use `*format`. `optional::operator*` is probably more appropriate here since `optional::value` first checks whether the optional has a value, but that check is done already.

Reply:
if (opt) { *opt } seems to be more formal and friendly way to our CIs